### ISSUE TITLE
receiving 404 error when authorizing service accounts

### ISSUE THREAD

**Issue by janiececald** [GAM-team/got-your-back#362]
The issue tracker is for reporting product deficiencies. How do I questions should be posted to the discussion forum at https://groups.google.com/group/got-your-back. When in doubt, start at the discussion forum and return here only when instructed to do so.

Please confirm the following:
* I have upgraded to the latest GYB release from https://github.com/jay0lee/got-your-back/releases and I still have this issue.
* I am typing the command as described in the GAM Wiki at https://github.com/jay0lee/got-your-back/wiki

Full steps to reproduce the issue:
1. When Checking service account DwD for scopes getting fail
2. try to launch https://gyb-shortn.jaylee.us/g2yau9
3. get a 404 error

Expected outcome (what are you trying to do?):
- Scope API page prompts, does not work
- 
<img width="591" alt="Screen Shot 2022-03-07 at 5 22 37 PM" src="https://user-images.githubusercontent.com/69479906/157147432-36a7dd30-eafd-46c7-91bc-df945eb6b642.png">

Actual outcome (what errors or bad behavior do you see instead?):
<img width="635" alt="Screen Shot 2022-03-07 at 5 19 16 PM" src="https://user-images.githubusercontent.com/69479906/157147398-1fc07056-69e1-42cf-a325-c18e321ff2d0.png">

**Comment by Porkstone** [comment]
I'm seeing this as well - Installed GYB today so using a very recent version

**Comment by janiececald** [comment]
I figured it out, the beginning of the link should read this "https://admin.google.com/ac/owl/domainwidedelegation?clientScopeToAdd= <https://admin.google.com/ac/owl/domainwidedelegation?clientScopeToAdd=>“

NOT :"https://admin.google.com/yourdomain.com/ManageOauthClients?clientScopeToAdd= <https://admin.google.com/b8ta.com/ManageOauthClients?clientScopeToAdd=https://mail.google.com/,https://www.googleapis.com/auth/apps.groups.migration,https://www.googleapis.com/auth/drive.appdata,https://www.googleapis.com/auth/userinfo.email&clientNameToAdd=116276825312893539343>"

**Comment by siliconpi** [comment]
So this can only be done by the domain administrator account? I can't get access to this page.

**Comment by jay0lee** [comment]
Correct, if you are not the admin you cannot use service account authentication. That's by Google design, only admins have ability to authorize the service account and backup/restore mail data of other users.

You should just use the normal authentication as your own user.

Jay

**Comment by siliconpi** [comment]
I'm just following the instructions in the program. There doesn't appear to
be anything I need to change in case I'm a normal user.

**Comment by siliconpi** [comment]
Sorry - to be clearer, I'm just an ordinary user (within a Google Workspace domain) trying to export my data out. I don't have any admin privileges. I followed the instructions and got to the screen shown by @janiececald but cannot access that admin page.

I'm not clear on what the "the normal authentication as your own user" is supposed to be.

You mean I should just use Takeout, or something else? I have the Takeout MBOX file. Will GAM be compatible with that?
